maestro-client/src/main/java/maestro/mockserver/MockInteractor.kt

```diff
@@ -46,6 +46,7 @@ class MockInteractor(
         .build()
 
     fun getCachedAuthToken(): String? {
+        if (!System.getenv("MAESTRO_CLOUD_API_KEY").isNullOrEmpty()) return System.getenv("MAESTRO_CLOUD_API_KEY")
```

Comment:
Nit: you can use `EnvUtils.maestroCloudApiKey()` here

Reply:
EnvUtils is in another package and it would need to add it as a dependency, that's why didn't do it.